Initial configurations for SFP management port LED

Copyright (C) 2019 Accton Technology Corporation

SPDX-License-Identifier:     GPL-2.0

diff --git a/drivers/net/ethernet/intel/igb/e1000_82575.c b/drivers/net/ethernet/intel/igb/e1000_82575.c
index 4a50870..2efdacb 100644
--- a/drivers/net/ethernet/intel/igb/e1000_82575.c
+++ b/drivers/net/ethernet/intel/igb/e1000_82575.c
@@ -301,6 +301,12 @@ static s32 igb_init_phy_params_82575(struct e1000_hw *hw)
 				hw->mac.ops.check_for_link =
 						igb_check_for_link_media_swap;
 		}
+		if (phy->id == M88E1112_E_PHY_ID) {
+			phy->ops.write_reg(hw, E1000_M88E1112_PAGE_ADDR, 0x3);
+			phy->ops.write_reg(hw, E1000_M88E1512_CFG_REG_1, 0x7204);
+			phy->ops.write_reg(hw, E1000_M88E1512_CFG_REG_2, 0x4455);
+			phy->ops.write_reg(hw, E1000_M88E1112_PAGE_ADDR, 0);
+		}
 		if (phy->id == M88E1512_E_PHY_ID) {
 			ret_val = igb_initialize_M88E1512_phy(hw);
 			if (ret_val)
